docs/_articles/en/index.md

```diff
@@ -10,14 +10,14 @@ lang: en
 </div>
 <br>
 
-## Salesforce CLI Scanner Plug-in
+## Salesforce Code Analyzer Plug-in
 
-The Salesforce CLI Scanner plug-in is a unified tool for static analysis of source code, in multiple languages
+The Salesforce Code Analyzer plug-in is a unified tool for static analysis of source code, in multiple languages
 (including Apex), with a consistent command-line interface and report output. We currently support the
 [PMD rule engine](https://pmd.github.io/), [PMD Copy Paste Detector](https://pmd.github.io/latest/pmd_userdocs_cpd.html), [ESLint](https://eslint.org/), and [RetireJS](https://retirejs.github.io/retire.js/).
 We may add support for more rule engines in the future.
 
-The Salesforce CLI Scanner Plug-in creates "Rule Violations" when the scanner identifies issues. Developers use this information as feedback to fix their code. 
```

Comment:
Not this line, but line 36 in this file has a reference to "Scanner". Putting comment here since GitHub doesn't let me add this in the correct location.